front/app/(visualiser)/interpeller/page.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+
+import SearchBar from '@/components/SearchBar/SearchBar';
+import { buttonVariants } from '@/components/ui/button';
+import { ChevronRight } from 'lucide-react';
+
+export default function Page() {
+  const router = useRouter();
+  const displayTransparencyScore = async (siren: string) => {
+    router.push(`/interpeller/${siren}/step1`);
+  };
```

Comment:
C est ce que je voulais dire ! C est plus clair pour exprimer ce que ca fait
```suggestion
  async function goToStep1(siren: string) {
    router.push(`/interpeller/${siren}/step1`);
  }
```